accepted/2024/end-of-life-packages.md

```diff
@@ -0,0 +1,110 @@
+Today, there exists two major package status mechanisms for understanding if a package is [vulnerable](https://learn.microsoft.com/en-us/nuget/concepts/auditing-packages) and/or [deprecated](https://learn.microsoft.com/en-us/nuget/nuget-org/deprecate-packages).
+
+Recently, the NuGet team has shipped functionality allowing package authors to include markdown READMEs to contain a hodgepodge of information relating to the package such as installation instructions, getting started examples, contributing tips, and less often seen is how the package is supported through an official or unofficial support policy.
+
+To make these package statuses clearer to package consumers and authors, this proposal introduces a new concept to indicate when a package has reached the end of its intended life.
+
+The reason being is that the current package statuses mean specific things today and how they are perceived such as:
+
+- Vulnerable - Indicates the package has a known security vulnerability registered in an official CVE database.
+- Deprecated - Indicates the package is deprecated, meaning that people should consider migrating off of the package for a common reason such as it is **legacy, included critical bugs, or any other reason** the maintainer wants to disclose. These packages are no longer recommended to use but may still receive updates.
+
+Introducing a concept of **"end of life"** will make it clear to package authors that these packages are not intended to be used any further as they are no longer supported, will not receive critical updates, and feedback will not be considered. This would be a new definition including:
+
+- End of life (EOL) - Indicates the package is no longer maintained, supported, or updated by its authors.
+
+| Status      | Definition                              | Updates Available | User Guidance                          | Risk     |
+|-------------|-----------------------------------------|-------------------|----------------------------------------|----------|
+| EOL         | No longer maintained or supported       | No                | Migrate to alternatives                | High     |
+| Vulnerable  | Contains known security vulnerabilities | Possibly          | Update or find secure alternatives     | High     |
+| Deprecated  | Not recommended for use                 | Possibly          | Transition to newer packages           | Moderate |
+
+## Motivation 
+
+<!-- Why are we doing this? What pain points does this solve? What is the expected outcome? -->
+
+There are two primary motivations in doing this work. The first being a direct answer to the "am I supported?" question. This is especially prevalent in the .NET ecosystem due to many first-party package providers have robust support policies that require a developer to understand with a decoder ring and too much time on their hands. It should be explictly simple to know you're supported in your current solution so long as your packages do not report EOL.
```

Comment:
> first-party package providers

I don't know what this term means. Are there both first-party and third-party package providers? Perhaps use just "package providers"?